packages/react-instantsearch-hooks-dom/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-instantsearch-hooks-dom",
+  "name": "react-instantsearch-hooks-web",
```

Comment:
Will it work without changing the name of the directory (`packages/react-instantsearch-hooks-dom`) as well?